### ISSUE TITLE
add jurelmetal as repo admin

### ISSUE THREAD

**Issue by cgallegu** [cgallegu/semantle-es#7]
@jurelmetal tell me how to do it and I'll do it. Couldn't find how (in the 30ms I allocated for it xD).

**Comment by jurelmetal** [comment]
Looks like Github doesn't support having more than one repo admin - you would need to transfer ownership, which is not what we want (I think).

https://stackoverflow.com/questions/8015579/add-another-user-to-project-owners-in-github

**Comment by cgallegu** [comment]
Yeah, and the alternative of creating an Organization just for this seems
overkill.

I updated the branching config (which is how this conversation got started).

Let's see if we keep on running into problems in the future and we can
evaluate creating an org for this.

Hopefully the repo settings will converge to something stable soon.

_Unless_ having those permissions is something whoever ends up setting up
CI/CD needs. If that's the case we can do the org stuff as part of that
work.
